recipes/gatspy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gatspy" %}
+{% set version = "0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "5bb9acf524d3583985cf689f7643fb8cb380f20387b6f904b77b468bbaf982bf"
+
+build:
+  number: 0
```

Comment:
This can also be `noarch: python` (see https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python)